### DISCUSSION TITLE
kmenu not launching

### DISCUSSION THREAD

**Discussion by pareekadi** [harshhhdev/kmenu#4] [Q&A] [answered]
I was setting up kmenu and it wasn't showing. I was using the same code from the documentation.  Here is my code:

```jsx
const Palette = () => {
  const [open, setOpen] = useState(0)
  return <Palette open={open} setOpen={setOpen} index={1} commands={commands} categories={["Main"]} />
}
```

**Comment by haaarshsingh** [comment, marked as answer]
I can see two issues here:

 - The command palette that you're opening is not marked as `main`. Remember, the command palette you'd want the user to first see when they click the shortcut should have main set to true.
 - No commands are defined in the file. They could've been defined outside the function, but I don't see them.
